### ISSUE TITLE
Container start failing after adding network to container

### ISSUE THREAD

**Issue by mhagen-vmware** [vmware/vic#1235]
Steps:
```
 2086  docker -H 192.168.254.194:2375 network create test-network
 2087  docker -H 192.168.254.194:2375 network ls
 2088  docker -H 192.168.254.194:2375 create busybox ifconfig
 2089  docker -H 192.168.254.194:2375 network connect test-network 52174935373d5b0f6ed9a9d8dc8bdc366c43da6cb32f777e93dfc16861be481a
 2090  docker -H 192.168.254.194:2375 start 52174935373d5b0f6ed9a9d8dc8bdc366c43da6cb32f777e93dfc16861be481a
```
Response:
```
Error response from daemon: server error from portlayer
Error: failed to start containers: 52174935373d5b0f6ed9a9d8dc8bdc366c43da6cb32f777e93dfc16861be481a
```
Snip from the container log:

```
INFO[0000] Loading main configuration                   
INFO[0000] Loaded executor config                       
INFO[0000] Added IP address for bridge: 172.16.0.2      
INFO[0000] Added hosts entry: 172.16.0.2 bridge.localhost 
ERRO[0000] failed to apply network endpoint config: unable to assign IP for net test-network: failed to add address to test-network: numerical result out of range 
INFO[0000] Shutting down reaper                         
INFO[0000] Stopping extension Attach 
```
[52174935373d5b0f6ed9a9d8dc8bdc366c43da6cb32f777e93dfc16861be481a.debug.zip](https://github.com/vmware/vic/files/335823/52174935373d5b0f6ed9a9d8dc8bdc366c43da6cb32f777e93dfc16861be481a.debug.zip)

[container-logs.tar.gz](https://github.com/vmware/vic/files/335818/container-logs.tar.gz)

**Comment by hmahmood** [comment]
Maybe related to #1214

**Comment by andrewtchin** [comment]
Tested this against vCenter and it appears to be resolved (5bb92e7)

```
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls pull busybox
Using default tag: latest
latest: Pulling from library/busybox
a3ed95caeb02: Pull complete 
8ddc19f16526: Pull complete 
Digest: sha256:65ce39ce3eb0997074a460adfb568d0b9f0f6a4392d97b6035630c9d7bf92402
Status: Downloaded newer image for library/busybox:latest
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls network create test-network
9802348a9627af3c3db348ad70267eee
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls network ls                 
NETWORK ID          NAME                DRIVER
19fe3353c47d        bridge              bridge              
9802348a9627        test-network        bridge              
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls create busybox ifconfig
9be52bb054a4fb06a0db92c34e89464759f785a426289b7ec96fa7485c27846a
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls network connect test-network 9be52bb054a4fb06a0db92c34e89464759f785a426289b7ec96fa7485c27846a
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls start 9be52bb054a4fb06a0db92c34e89464759f785a426289b7ec96fa7485c27846a                      
9be52bb054a4fb06a0db92c34e89464759f785a426289b7ec96fa7485c27846a
chin@CORPORATE:~/go/src/github.com/vmware/vic|1347/cli-args 
⇒  docker -H 10.17.109.167:2376 --tls ps -a                                                                 
CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
9be52bb054a4        busybox             "ifconfig"          About a minute ago   Stopped                                 ovial_ramanujan
```
